src/workerd/api/tests/js-rpc-flag.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+import assert from 'node:assert';
+
+// This class does not extend `DurableObject`, but because we have the `js_rpc` compat flag on,
+// it'll still accept RPC.
```

Comment:
You deleted the "js-rpc-disabled" test, should something like that still exist?

Reply:
As of this commit, the main `js-rpc-test` no longer has the flag set, and tests all the behavior around that, making `js-rpc-disabled` redundant. I did however introduce `js-rpc-flag` which is the opposite test: it tests behavior when the flag is _enabled_.